packages/update-peers.js

```diff
@@ -0,0 +1,11 @@
+const fs = require ('fs');
+
+const data = fs.readFileSync('package.json');
+const content = JSON.parse(data.toString());
+content.peerDependencies = content.peerDependencies || {};
+for (let p of Object.getOwnPropertyNames(content.devDependencies)) {
+  if (p.startsWith('@aurelia-ux')) {
+    content.peerDependencies[p] = content.devDependencies[p];
```

Comment:
@MaximBalaganskiy are we sure that after pumping the version of a package, lerna also update the dev dependencies versions? or it waits until every package is done before bumping this? If it's the later, we may have issue here

Reply:
Lerna does packages one by one calling appropriate hooks. I've tried this in my local branch, works as intended.

Reply:
Ok that's awesome. Thanks for this PR. let's go then